packages/bits/spec/virtual-camera/EyesLens.ts

```diff
@@ -2,7 +2,7 @@ import { ProtractorBrowser } from "protractor";
 import { ICameraSettings, ILens } from "./types";
 import { getCurrentBranchName } from "../helpers";
 export class EyesLens implements ILens {
-    private eyes: any;
+    public eyes: any;
 
     constructor(private browser: ProtractorBrowser, private settings: ICameraSettings) {
```

Comment:
Maybe `cameraSettings` to make it a bit clearer what it is when you're reading another part of the file:

```suggestion
    constructor(private browser: ProtractorBrowser, private cameraSettings: ICameraSettings) {
```